dist/api_data.py

```diff
@@ -479,6 +510,8 @@ def __ge__(self, other):
 ]
 
 index_only_config = [
+    Config('extractor', 'none', r'''
```

Comment:
I don't see this being sued in WT? Do you mean this was present in the older versions? and it causes issues when customers upgrade to the new version?

Reply:
Yeah, it was present in older versions. It only causes problems if customers explicitly set it, even to empty (e.g. `extractor=,`). Unfortunately there used to be a backup tool that would explicitly set a bunch of config settings to the empty string, and some of these clusters are still around. So this needs to hang around forever.

Reply:
That's right, these metadata entries have already been created and are out in the wild. We could author a tool that removed them, but I've seen some scary numbers about how many clusters are affected (six figures).